tools/benchmark

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+LEGOFY=$(pwd)/bin/legofy;
+PATH_IMAGES=$(pwd)/images/benchmark/;
+
+benchmark() {
+  local flags=$@;
```

Comment:
Converting "array" to string, may lead to unexpected behavior if a single argument contains spaces because it will be interpreted as multiple arguments down the way. Not a problem with what the options currently look like but I'd rather warn about this.

I'd rather:

```sh
default_flags='--filter-quality=80 --pattern-quality=80 --quality=good'
$LEGOFY "${@:-$default_flags}" ...
```

This way `$@` will be properly expended.